src/dependency-manager/src/service-config/v1.ts

```diff
@@ -246,7 +246,7 @@ export class ServiceConfigV1 extends ServiceConfig {
   environment?: Dictionary<string>;
 
   @IsOptional({ groups: ['operator', 'debug'] })
-  @IsObject({ groups: ['developer'], message: 'interfaces must be defined even if it is empty since the majority of services need to expose ports' })
+  @IsObject({ groups: ['developer'], message: 'interfaces must be defined even if it is empty since the majority of services need to expose ports' }) //TODO:84: this assumption becomes less applicable with tasks
```

Comment:
Maybe we have a base config that is shared between task/service? Then we can define Service specific fields at a higher level. This would probably be a nightmare for versioning. @davidthor  thoughts?